src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -166,10 +158,12 @@ public Object createModule(String name) throws CheckstyleException {
                 instance = createObjectFromMap(name, thirdPartyNameToFullModuleNames);
             }
         }
-
         if (instance == null) {
             instance = createObject(name);
         }
+        if (instance == null && useBruteForceModuleLoading) {
+            instance = createModuleByBruteForce(name);
```

Comment:
when I read this code .... I do not understand what is brute force mean here.... it is naming desing problem.
`searchModuleInAllPackages ... createModuleByTryInEachPackage(packages, name)` ?

Reply:
renamed